cluster/examples/kubernetes/ceph/operator.yaml

```diff
@@ -149,6 +149,10 @@ spec:
         - name: ROOK_DISABLE_DEVICE_HOTPLUG
           value: "false"
 
+        # Provide customised regex as the values using comma. If 'nil', default values will be used.
```

Comment:
It'd be nice to give an example in the comments.

Reply:
Ok. I will provide an example.